docs/pages/setup/reference/moderated-sessions.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Moderated Sessions
+description: Moderated Sessions RBAC Reference
+h1: Moderated Sessions
+---
+
+# Moderated Sessions
+
+Moderated Sessions makes use of RBAC to control access to and within sessions.
+Here is a collection of details on configuring Moderated Sessions.
+
+## RBAC
+
+Moderated Sessions makes use of RBAC policies to allow for fine grained control over
+who can join a session and who are required to be present to start one.
+
+The system is based around **require policies** and allow policies. Require policies define a set of conditions
+that must be a met for a session to start/run.
+
+**Allow policies** are used to define what sessions a user can join
+and under what conditions they may join a session.
+
+### Session kinds
+
+Require and allow policies have to specify which sessions they apply to. Valid options are `ssh` and `k8s`.
+
+### Filters
+
+A filter expression or predicate is a string statement used to define logic based on a set of input variables.
+These follow common programming language syntax and allow for more detailed control.
+`user.name == "adam" || contains(user.traits, "view")` is a simple example of a filter expression
+that evaluates to true if the user is Adam or if the user has the trait `view`.
+
+Require policies can specify which users they consider as valid with a filter expression. They work like
+other filters used within Teleport such with access requests. The filter context has a `user` object defined
+which the set fields `roles`, `traits` and `name`.
```

Comment:
While I know a full example is coming up in this page, I think other readers may be expecting an example here. Otherwise, it may not be clear where a filter appears within a config resource.

Reply:
Do you mean just a filter example or like a minimal policy with a filter? The reason I am hesitant is that it'll be pretty much the exact same as the examples below since there aren't really optional fields on a policy.